fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/observer/SimpleObserver.java

```diff
@@ -68,6 +82,60 @@
     static final Integer EVENT_TYPES = NODE_ADDED + NODE_REMOVED + NODE_MOVED + PROPERTY_ADDED + PROPERTY_CHANGED
             + PROPERTY_REMOVED;
 
+    private static Stream<FedoraResource> getAllChildren(final FedoraResource resource) {
```

Comment:
Yeah, that allow a simple migration with a `default` impl for the current method. Are there other cases? I.e. There's recursive/non-recursive, are there other characteristics we need to encode?